### PR TITLE
Email and PAN addition in ECREQ as optional parameter

### PR DESCRIPTION
**Pull request by Athira-Jayakumar1**
Email and PAN addition in ECREQ as optional parameter

### CODE REVIEW

**Comment by Athira-Jayakumar1** [comment]
@dheerajkhardwal @TeamSahamati Please review and approve

**Comment by dheerajkhardwal** [comment]
Marking this to be released as `v1.3.0`

**Review by dheerajkhardwal — APPROVED** [review]

File: specification/request-specification.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ Below are the required parameters that will be encrypted using AES256 encryption
...
 | srcref             | Array              | Array of consent handle id(s), as returned by AA server to the /Consent request api invoked by the FIU(s) on the AA prior to this redirection call.                                                                                                                                                                                        |
 | fipid              | Array              | Optional, can be blank. If FIU already has the information that citizen wants to discover accounts of one or more FIP(s) beforehand, then they can pass this info to AA. It saves user clicks and enhances journey experience                                                                                                              |
+| emailid            | String              | Optional, can be blank. If FIU wants to notifiy the consent creation and subsequent notifications to the citizen thorugh email
```
Comment (dheerajkhardwal) [inline specification/request-specification.md:32]: Small typo in the description text: `thorugh`, please update.
Also, shall we keep the field name as just `email`?

File: specification/request-specification.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ Below are the required parameters that will be encrypted using AES256 encryption
...
+| PAN              | String              | Optional, can be blank. PAN of the citizen for discovery of certain FI categories like Mutual Funds
```
Comment (dheerajkhardwal) [inline specification/request-specification.md:33]: In order to keep the field naming consistent, let's use lowercase for this field name as well.

**Review by dheerajkhardwal — CHANGES_REQUESTED** [review]

**Comment by Athira-Jayakumar1** [comment]
@dheerajkhardwal updated, Please check

**Review by dheerajkhardwal — APPROVED** [review]

**Comment by dheerajkhardwal** [comment]
Thanks @Athira-Jayakumar1.

**Pull request merged**